entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md

```diff
@@ -3527,3 +3549,783 @@ Notice:
 - The iterator stops traversing from a given node when the callback delegate returns `false`. This example keeps track of visited entities and returns `false` when the entity has already been visited. This prevents infinite loops resulting from cycles in the graph.
 - The `EntityEntryGraphNode<TState>` object allows state to be passed around without capturing it into the delegate.
 - For every node visited other than the first, the node it was discovered from and the navigation is was discovered via are passed to the callback.
+
+## Model building enhancements
+
+EF7 contains a variety of small improvements in model building.
+
+> [!TIP]
+> The code for samples in this section comes from [ModelBuildingSample.cs](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Miscellaneous/NewInEFCore7/ModelBuildingSample.cs).
+
+### Indexes can be ascending or descending
+
+By default, EF Core creates ascending indexes. EF7 also supports creation of descending indexes. For example:
+
+```csharp
+            modelBuilder
+                .Entity<Post>()
+                .HasIndex(post => post.Title)
+                .IsDescending();
```

Comment:
The main reason to use descending indexes IIRC is for composite indexes, since a single-column index can be traversed either way anyway; maybe it's good to mention that and have the basic example be composite?

Reply:
I did not realize this! Thanks!

Reply:
(Should probably confirm this across database at some point! But it makes sense...)